src/services/service-data.ts

```diff
@@ -2,6 +2,9 @@ import { toReactive } from "@vueuse/core";
 import useSWRV from "swrv";
 import { reactive, computed, watch } from "vue";
 
+const refreshInterval =
+  Number.parseInt(import.meta.env.VITE_API_REFRESH_INTERVAL_MS, 10) || 0;
```

Comment:
wat  gebeurd er als de interval 0 is?

Reply:
dan wordt er niet gepollt. maar is hier undefined in plaats van 0 zetten explicieter?

Reply:
dat vind ik wel, ja.